.github/workflows/e2e-stress-test-flake-fix.yml

```diff
@@ -66,9 +76,9 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
           maildev: true
           openldap: true
-          postgres: true
-          mysql: true
-          mongo: true
+          postgres: ${{ inputs.qa_db == 'postgres' }}
+          mysql: ${{ inputs.qa_db == 'postgres' }}
```

Comment:
Aaaah, I see what you did for mysql now.
But why did you decide to not add mysql to the list of options separately?

Reply:
answered in here. https://github.com/metabase/metabase/pull/46065#discussion_r1691163803 tl;dr I think we could make the option name more obvious that it also includes mysql